components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go

```diff
@@ -92,14 +86,14 @@ func (o *options) String() string {
 	return fmt.Sprintf("--proxyPort=%d --externalAPIPort=%d --tenant=%s --group=%s "+
 		"--eventingPathPrefixV1=%s --eventingPathPrefixV2=%s --eventingPublisherHost=%s "+
 		"--eventingPathPrefixEvents=%s --eventingDestinationPath=%s "+
-		"--appRegistryPathPrefix=%s --appRegistryHost=%s --appName=%s "+
+		"--appName=%s "+
```

Comment:
Be aware that removing this from the application's code is not enough. It should be removed from deployment.